### ISSUE TITLE
Add a new open source programme in SOC 

### ISSUE THREAD

**Issue by yusra05** [Sriparno08/Openpedia#197]
## Category

- [ ] Documentation
- [x] Resource Addition
- [ ] Codebase
- [ ] User Interface
- [ ] Feature Request

## Description

I wanted to add some more open source programmes in SOC section
- XROS

## Open-source
I'm contributing under JWOC

**Comment by Sriparno08** [comment]
Hey @yusra05, you can only add one resource for an issue. If you want to add more, you need to open issues for each of them.

Also, please include the 'JWOC' (Case-Sensitive) keyword in your comment to be considered for assignment.

**Comment by Ananyamishra08** [comment]
hello @Sriparno08 i would like to add one open source programme in SOC section as well , Assign this to me under JWOC

**Comment by Sriparno08** [comment]
@Ananyamishra08 You're already assigned to another issue. Once your PR for that issue is merged, you will be considered for assignment on other issues.

**Comment by Ananyamishra08** [comment]
I haved completed that task . Kindly check and merge my PR .

**Comment by yusra05** [comment]
@Sriparno08 I have updaed my comment.
Now assign this issue to me

**Comment by Divyanshgarg-04** [comment]
@Sriparno08 I would love to work on this issue can you please assign this issue to me

**Comment by Sriparno08** [comment]
@yusra05 You're assigned.

To Others:

The mentee who is assigned to this issue is chosen because he/she was the first one to ask for assignment with the 'JWOC' keyword in his/her comment.
